projects/kit/src/lib/time/processors/zero-placeholders-preprocessor.ts

```diff
@@ -0,0 +1,21 @@
+import {MaskitoOptions} from '@maskito/core';
+
+export const zeroPlaceholdersPreprocessor: NonNullable<
```

Comment:
Just a thought: I wonder if we would need to add `pre/postprocessorPipe` utility function to pipe multiple processors in order.